portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path

```diff
@@ -60,7 +60,7 @@
 </tr>
 <tr>
 	<td>ANY</td>
-	<td>//button[contains(.,'${key_text}')]</td>
+	<td>//a[contains(.,'${key_text}')] | //button[contains(.,'${key_text}')]</td>
```

Comment:
did you try something like ?
//*[contains(.,'${key_text}') and contains(@class, 'btn')]

Reply:
Yes, I tried, but not all buttons have class 'btn', that's why we updated this locator to https://github.com/liferay/liferay-portal/blob/master/portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Button.path#L66-L70 before.

And if use //*[contains(.,'${key_text}')], there will be more than one elements match it in some pages, but I didn't find out a common class or something else in both kinds of buttons (//button or //a).

Reply:
My changes caused tests failed at sign in. I will try the path that you mentioned, and will create a new locator for the button that doesn't have 'btn' class. Thanks for the help!